interfaces/api/v1/engine/autopilot_routes.py

```
@@ -760,7 +760,7 @@ async def stream_debug(novel_id: str):
             sample_msg = queue.get_nowait()
             # 把消息放回去
             queue.put(sample_msg)
-        except:
+        except Exception:
             pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)